### PR TITLE
No index old standards

### PR DESCRIPTION
**Pull request by odscjames**

### CODE REVIEW

**Comment by odscjames** [comment]
https://developers.google.com/search/docs/crawling-indexing/sitemaps/build-sitemap says ` Google ignores <priority> and <changefreq> values. `

So don't index old ones?

Let's discuss how further to go here.

At least we probably want the "203" string to be a Django Setting somewhere.

Do we want to try and have the version as a field on the page somewhere so we aren't parsing URLs? (Not that the URL's are likely to change.) Should we try and make guidance dev docs a different model?

Or is not indexing a bad idea in all, what if people want to find old versions? Then I think our only option is to make a giant big banner on the top of old versions saying "Go here for latest version!" (Maybe we should do that anyway?)

**Comment by odscjames** [comment]
TODO: Just realised we'll want to remove these pages from the site map too.

**Comment by odscjames** [comment]
Last TODO done and now ready for review @Bjwebb

File: iati_standard/models.py [hunk 1]
```diff
@@ -397,10 +399,35 @@ def save(self, *args, **kwargs):
...
+
+    def get_sitemap_urls(self, request=None):
+        """If it's an older version of the standard, we don't want it indexed."""
+        if self.is_older_version_of_standard():
+            return []
+        else:
+            return [
+                {
+                    'location': self.get_full_url(request),
+                    'lastmod': (self.last_published_at or self.latest_revision_created_at),
```
Comment (Bjwebb) [inline iati_standard/models.py:427]: Are these lastmod times updated each time the standard site is deployed? (Even if there's no change to the individual page).

Comment (odscjames) [reply iati_standard/models.py:427]: Live site:
```

<url>
<loc>https://iatistandard.org/en/iati-standard/203/</loc>
<lastmod>2024-03-22</lastmod>
</url>
<url>
<loc>
https://iatistandard.org/en/iati-standard/203/activity-standard/
</loc>
<lastmod>2024-03-22</lastmod>
</url>
```

Local dev:
```

<url>
<loc>https://iatistandard.org/en/iati-standard/203/</loc>
<lastmod>2024-03-22</lastmod>
</url>
<url>
<loc>
https://iatistandard.org/en/iati-standard/203/activity-standard/
</loc>
<lastmod>2024-03-22</lastmod>
</url>
```

**Review by Bjwebb — APPROVED** [review]

**Pull request merged**